org.hedgewars.Hedgewars.yaml

```diff
@@ -0,0 +1,399 @@
+app-id: org.hedgewars.Hedgewars
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: hedgewars
+rename-desktop-file: hedgewars.desktop
+rename-icon: hedgewars
+rename-appdata-file: hedgewars.appdata.xml
+finish-args:
+- --socket=x11
+- --share=ipc
+- --socket=wayland
+- --socket=pulseaudio
+- --device=all
+- --share=network
+- --persist=.hedgewars
+modules:
+- shared-modules/glew/glew.json
+- shared-modules/glu/glu-9.0.0.json
+- name: lua51
+  sources:
+  - type: archive
```

Comment:
Have you considered LuaJIT? https://github.com/flathub/io.github.GnomeMpv/blob/master/org.luajit.Luajit.json

Reply:
Doesn't seem to work.